### PR TITLE
PCBC-987: Fix consistency vector encoding for FTS

### PR DESCRIPTION
**Pull request by avsej**

### CODE REVIEW

File: Couchbase/SearchOptions.php [hunk 1]
```diff
@@ -322,7 +313,7 @@ public static function export(?SearchOptions $options): array
             'disableScoring' => $options->disableScoring,
             'fields' => $options->fields,
             'sortSpecs' => $sort,
-            'consistentWith' => $options->consistentWith,
+            "consistentWith" => $options->consistentWith == null ? null : $options->consistentWith->export(),
```
Comment (thejcfactor) [inline Couchbase/SearchOptions.php:316]: Seems like single-quotes are the norm...

**Review by thejcfactor — DISMISSED** [review]
Just a nitpick -- LGTM otherwise (not a breaking change b/c the use of consistency w/ FTS is broken as-is).

File: Couchbase/SearchOptions.php (hunk 1 above)
Comment (avsej) [reply Couchbase/SearchOptions.php:316]: thanks. I will run prettyfier

**Review by thejcfactor — APPROVED** [review]
LGTM 👍

**Pull request merged**